[Feat] #662 앱잼탬프 스탬프 상세 조회 API 구현

jher235 opened the pull request:
## Related issue 🛠

<!-- 관련 이슈 번호를 적어주세요 -->

- closes #662

## Work Description ✏️
- 앱잼탬프 스탬프 상세 조회 API 구현 
- 네이밍 통일 

<!-- 작업 내용을 간단히 소개주세요 -->

## To Reviewers 📢
- 네이밍 변경 관련은 논의한 대로 아래와 같이 변경했습니다.
  -  TeamInfo -> AppjamUser
  - mission -> AppjamMission 
  - stamp || soptamp -> Appjamtamp

- 기존에 Stamp, SoptampUser 관련 작업을 SoptamtFacade 에서 처리하고 있어서 AppjamtampFacade를 새로 두었습니다. read는 AppjamtampFacade, write은 SoptampFacade를 사용하고 있어서 통일성이 좀 떨어지는 것 같긴 하네요..
<!-- 리뷰어들에게 물어볼 점, 할 말 등을 적어주세요 -->

jher235 committed:
[#662] refactor: 네이밍 변경 및 AppjamUserService 로직 분리
TeamInfo -> AppjamUser
mission -> AppjamMission
stamp || soptamp -> Appjamtamp

jher235 committed:
[#662] feat: userId 로 팀 정보를 반환하는 메서드 추가

jher235 committed:
[#662] feat: 앱팀탬프 스탬프 상세 조회 로직 구현

jher235 committed:
[#662] feat: 앱팀탬프 스탬프 상세 조회 presentation layer 구현

huncozyboy commented on src/main/java/org/sopt/app/application/stamp/StampInfo.java:
```diff
@@ -67,4 +69,55 @@ public static StampView of(Stamp stamp, int myClapCount, boolean isMine, String
+        private LocalDateTime updatedAt;
+        private int clapCount;
+        private int viewCount;
+        private boolean isMine;
```
`isMine` 필드를 추가로 반환해주는 이유가 있을까요 ??

huncozyboy reviewed (approved):
고생하셨습니다 ! 
테이블이랑 엔드포인트 네이밍 관련해서는 이미 논의된 내용이라서 따로 리뷰로 남기진 않았습니당

jher235 committed:
[#662] refactor: getTeamSummary 메서드에 파라미터 명시
- 기존에는 파라미터 타입이 겹칠일이 없을 것 같아서 오버로딩으로 목적에 집중해서 사용할 수 있도록 구현했음. 그러나 추후 확장성을 고려하여 by~ 의 형태로 네이밍을 수정
